grammar.pegjs

```diff
@@ -88,8 +88,12 @@ concatenation "concatenation of strings and/or variables"
           / commandSubstitution
           / singleQuote
           / doubleQuote
+          / equal
           )+
 
+equal "equal"
```

Comment:
I think we have to includes all characters allowed here. There are more than the = sign